src/games/utils/index.ts

```diff
@@ -67,7 +67,7 @@ export function adminAction(userRequesting: User): boolean {
   return !!(
     userRequesting.isSuperAdmin() ||
     userRequesting.isAdmin() ||
-    userRequesting.isCommunityTeam()
+    userRequesting.isGameDev()
```

Comment:
I exepct to completey remove this functions , but will leave it for another pr since it will require a deeper look into Slack commands